_posts/2019-05-28-experience-and-opinions-developing-for-google-assistant-and-amazon-alexa.md

```diff
@@ -0,0 +1,92 @@
+
+### Location
+{: .text-left }
+
+The Google Assistant and Alexa simulators are very nice. They give you screens to test presentation markup as well as other features. One feature that the Alexa simulator is missing is location. You cannot set a location for the simulator to use. Google makes this very simple and it is on the main UI. A major difference between the two assistants is that addresses are assigned to devices in Amazon. Amazon is thinking, where do I ship you all of your Holiday packages. The Google Assistant is on the every Android phone so location is where am I now.
+
+## Resources
+{: .text-left }
+
+### Documentation
+{: .text-left }
+
+There are so many pages of documentation written for the Google Assistant and Amazon Alexa. It is a massive undertaking. There are multiple versions of the same documentation for each SDK language and then the underlying request response contracts if you decide to roll your own. With that said, I found Alexa's documentation and demos to be better organized and more current. The ecosystem seemed to be treated more as one item. Google seemed to be offering different products that worked together and the documentation did not seem as cohesive. A lot of the demo code was on older versions of their SDK's.
+
+### Real Time Collaboration
+{: .text-left }
+
+Both digital assistants have slack channels.
+
+- `[amazonalexa.slack.com](https://amazonalexa.slack.com/)`
+- `[googleathome.slack.com](https://googleathome.slack.com/)`
+
+The Alexa slack channel is awesome. There are a lot of knowledgeable people in there chatting about building skills. Alexa has employees participating in the conversation. The Google channel is the opposite. A lot of questions go unanswered.
+
+Alexa also runs "[office hours](https://www.twitch.tv/amazonalexa)" on twitch. If you have never used twitch, it is becoming a valuable developer relations outreach tool. _It's not only for gaming_. Alexa developers host an hour "show" where they answer questions watchers type in and show off new features. If you stump them, they get back to you later. This is as good as it gets without pair programming with an Amazon employee.
+
+## Wrap up
+{: .text-left }
+
+Developing software for digital assistants is fun. Is it the next mobile? I do not think so. Are developers making a lot of money with skills? No. Is one assistant better than the other? I don't think so. I very much enjoyed developing for Alexa but Googles natural language from Dialogflow seemed smarter.
```

Comment:
```suggestion
Overall, I very much enjoyed developing for Alexa, but Google's natural language from Dialogflow seemed smarter. Is one assistant better than the other? I don't think so. Are digital assistants the next mobile? I do not think so. Are developers making a lot of money with skills? No. But Google Assistant and Amazon Alexa are both great tools, and I enjoyed trying them out.
```

Reply:
So maybe this isn't much better, but I tried! Beginnings and endings of articles are difficult. lol